include/shape3d.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef SHAPE3DH
+    #define SHAPE3DH
+
+    struct Vector3;
+    struct Color;
+
+    /**struct Shape3D;
+     * @brief Shape3D object
+     */ 
+    class shape3D {
+        public:
+            /**
+             * @brief Construct a new shape3D object
+             */
+            shape3D();
+            /**
+             * @brief Destroy shape3D object
+             */
+            ~shape3D();
+
+            /**
+             * @brief Draw sphere 
+             * @param centerPos position of the center of the sphere
+             * @param radius raduis of the sphere
+             * @param color Color of the sphere
+             */
+            void DrawSphere(Vector3 centerPos, float radius, Color color);
```

Comment:
Class methods should not be capitalised:
`void drawSphere(Vector3 centerPos, float radius, Color color);`

Reply:
Done in this commit "MODIFIED: shape3D class in indie namespace, shape3D method in lower case"